mkdocs.yml

```diff
@@ -38,15 +38,18 @@ nav:
         - Configuring your development environment: "dev/env.md"
         - Configuring <code>img2mml</code>: "dev/eqn2mml.md"
         - Building documentation: "dev/docs.md"
-        - Test coverage reports: "coverage/index.md"
-        - Code2fn coverage reports: "coverage/code2fn_coverage/report.html"
         - Building docker images: "dev/docker.md"
         - Publishing an incremental release: "dev/creating-an-incremental-release.md"
         - Adding a new model: "dev/adding_new_model.md"
         - Adding a new tree-sitter frontend: "dev/adding_new_tree_sitter_frontend.md"
         - Generating code2fn model coverage reports: "dev/generating_code2fn_model_coverage.md"
         - Using code ingestion frontends: "dev/using_code_ingestion_frontends.md"
         - Using tree-sitter preprocessor: "dev/using_tree_sitter_preprocessor.md"
+    - Coverage:
+        - Code2fn coverage reports: "coverage/code2fn_coverage/report.html"
+        - TA1 Integration Dashboard: "coverage/ta1_integration_dashboard.md"
```

Comment:
If you'd prefer, the value for this can just be the external URL.  Something like this:

```yaml:
    - Coverage:
        - Code2fn coverage reports: "coverage/code2fn_coverage/report.html"
        - TA1 Integration Dashboard: "https://integration-dashboard.terarium.ai/TA1"
```

Example here: https://www.mkdocs.org/user-guide/configuration/#nav

Reply:
Oh that would be better! I'll push that change in! Thanks!